test/JDBC/expected/sys-parsename-dep-vu-prepare.out

```diff
@@ -0,0 +1,53 @@
+CREATE TABLE parsename_Employee (
+ EmployeeID INT PRIMARY KEY,
+ FirstName NVARCHAR(50),
+ LastName NVARCHAR(50),
+ HireDate DATETIME,
+ Salary MONEY
+);
+GO
+
+-- Insert some sample data
+INSERT INTO parsename_Employee(EmployeeID, FirstName, LastName, HireDate, Salary)
+VALUES (1, 'John', 'Doe', '2020-01-01', 50000),
+ (2, 'Jane', 'Smith', '2020-02-01', 60000),
```

Comment:
Should add a test case like : 

SELECT PARSENAME('AdventureWorksPDW2012.dbo.DimCustomer', 1) AS 'Object Name';  
SELECT PARSENAME('AdventureWorksPDW2012.dbo.DimCustomer', 2) AS 'Schema Name';  
SELECT PARSENAME('AdventureWorksPDW2012.dbo.DimCustomer', 3) AS 'Database Name';  
SELECT PARSENAME('AdventureWorksPDW2012.dbo.DimCustomer', 4) AS 'Server Name';  
GO  

and SELECT PARSENAME('AdventureWorksPDW2012.dbo.DimCustomer', 0) AS 'Server Name';   -> ilegal , but need to see what's going on for illegal ones